src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs

```diff
@@ -73,6 +73,8 @@ private static bool GetIsWindowsSubsystemForLinux()
             return false;
         }
 
+        public static bool IsIntMaxValueArrayIndexNotSupported => true;
```

Comment:
It is better to use positive names (XXXSupported instead of XXXNotSupported) for readability.

Reply:
I agree but xunit conditionals don't support negating syntax (you have Not everywhere)

Reply:
ok

Reply:
Nit, but I believe when we do this we consistently begin with `IsNot...`